src/main/webapp/populateData.js

```diff
@@ -90,12 +90,24 @@ function populateGmail() {
  * Populate Tasks container with user information
  */
 function populateTasks() {
-  // Get Container for Tasks content
-  const tasksContainer = document.querySelector('#tasks');
+  let fetchFrom;
+  const select = document.querySelector('#panel__task-list-select');
```

Comment:
Nice BEM naming. Just an FYI, you don't have to follow this for id's - it's more for classes. Not an issue though

Reply:
Oh okay, i'll take off the `panel__` to keep it consistent, thanks!

Reply:
`RESOLVED`